composer.json

```diff
@@ -19,7 +19,7 @@
         "symfony/http-kernel": "~2.2",
         "symfony/dependency-injection": "~2.2",
         "symfony/templating": "~2.2",
-        "symfony/locale": "~2.2",
+        "symfony/intl": "~2.7",
```

Comment:
We have to support SF 2.3.

BTW, the 2.3 test was working. The problem is  `prefer-lowest`. You just have to find the minimum 2.3 version to have to get tests working.